bsmetadata/train.py

```diff
@@ -297,7 +297,10 @@ def main(args: CFG) -> None:
             model, optimizer, dummy_dataloader, scheduler
         )
     else:
-        format_fn = lambda x: x
```

Comment:
@cccntu just for your reference, made the following changes to fix the code quality changes. Hope it's fine.